### ISSUE TITLE
ID 258954: [Screen reader -Simon – Value Mapping]: Heading hierarchy is not defined sequentially in the "Time series graphs / Edit Panel" Screen.

### ISSUE THREAD

**Issue by natydej** [grafana/grafana#66554]
**Steps to reproduce**:

**Actual Result**:

**Expected Result**

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by papidb** [comment]
Hi, is this an issue I can pick up?

**Comment by GunFood** [comment]
Is there any update to the issue?

**Comment by ashharrison90** [comment]
cannot repro this in the specific scenario anymore, believe it's been fixed. if that's incorrect, please feel free to leave a comment and we can discuss/reopen :)
